charts/jenkins/Chart.yaml

```diff
@@ -36,3 +40,5 @@ annotations:
       image: jenkins/inbound-agent:3107.v665000b_51092-5
     - name: backup
       image: maorfr/kube-tasks:0.2.0
+  artifacthub.io/category: "integration-delivery"
+  artifacthub.io/license: "Apache-2.0"
```

Comment:
Adds some additional information to our artifacthub info card. Thoughts on enabling the security graph? We do the same for docker images.

Reply:
sure

Reply:
Then someone just needs to enable it 😅